### PR TITLE
organisation comment

### PR DESCRIPTION
**Pull request by michaelmajgaard**

### CODE REVIEW

File: libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html [hunk 1]
```diff
@@ -113,4 +113,17 @@
...
+      <watt-label>{{ t("commentLabel") }}</watt-label>
+      <textarea
+        wattInput
+        class="textarea-comment"
+        [(ngModel)]="changes.comment"
+        (ngModelChange)="onModelChanged()"
+        #commentInputModel="ngModel"
+      ></textarea>
+    </watt-form-field>
+  </div>
```
Comment (dzhavat) [inline libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html:128]: question: the `feature-edit-organization` library doesn't contain any tests. The component must be refactored at some point as well because it uses a pattern that we don't encourage (i.e. the `emit` in `ngOnChanges` part). Adding overall tests and general refactoring feels a bit out of scope for the PR, so can you at least create an issue on your backlog to track that?

File: libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html [hunk 2]
```diff
@@ -113,4 +113,17 @@
       </watt-error>
     </watt-form-field>
   </div>
+
+  <div class="form-container">
+    <watt-form-field class="field-comment">
+      <watt-label>{{ t("commentLabel") }}</watt-label>
+      <textarea
+        wattInput
```
[the remark this answers is not in the document]
Comment (michaelmajgaard) [reply libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html]: @dzhavat changed it to a normal input field for now

File: libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html (hunk 1 above)
Comment (michaelmajgaard) [reply libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html:81]: I'll create a maintenance issue in our backlog on this. @dzhavat

**Review by dzhavat — APPROVED** [review]

File: libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (frontend-specialisten) [reply libs/dh/market-participant/feature-edit-organization/src/lib/market-participant-edit-organization/master-data/dh-market-participant-organization-master-data.component.html]: Just an FYI, I created #626 for fixing the styling of text-areas
